src/qibolab/runcards/tii_zcu111.yml

```diff
@@ -67,7 +67,7 @@ native_gates:
             MZ:
                 duration: 1000
                 amplitude: 0.5
```

Comment:
Why amplitude=0.5, we usually use 0.1

Reply:
Here the runcard is not really correct since I wasn't yet able to fully calibrate.

But, in general, with this driver we will have higher amplitude for the readout pulses since in the multiplexed firmware qick actually divides the amplitude by the number of multiplexed tones (that I hardcoded as 3). 
So, in this case, an amplitude of 0.5 corresponds to 0.5/3=~0.17 of the old driver